content/doc/administrate/tcp-redirections.md

```diff
@@ -54,4 +110,3 @@ the active TCP redirections to make sure you have the proper data.
 
 We'll use namespace "default" and port "42" for this example.
```

Comment:
It looks like there's some missing content here.

Reply:
You mean a bash window ? Or did i miss something